### PR TITLE
{tools}[foss/2020b] URANOS v1.07

### PR DESCRIPTION
**Pull request by DirectorSloan**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+sources = [{
+    'filename': 'uranos-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/mkoehli',
+        'tag': 'v%(version)s',
+        'repo_name': 'uranos',
+    },
+}]
+
+checksums = ['ff90a50e128f257dd0840153c8f6926820f7293ed9eab1c208ec53995ec5b856']
```
Comment (Micket) [inline easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb:27]: Checksum won't be stable like this. But we don't need to do this
```
source_urls = ['https://github.com/mkoehli/uranos/releases/download/v%(version)s/']
sources = [{'download_filename': 'URANOS.zip', 'filename': SOURCELOWER_ZIP}]
checksums = ['2d0cdef79f048ad0bcab72a3a9134a16c306ad4b7d25d47ecd875f411c18d310']
```
should work i think

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+version = '1.07'
+
+homepage = 'https://github.com/mkoehli/uranos/tree/main'
+description = """Ultra Rapid Adaptable Neutron-Only Simulation for Environmental Research,
+ is a Monte Carlo toolkit specifically tailored for environmental sciences."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True}
+
+buildopts = 'CXXFLAGS="$CXXFLAGS -std=c++17"'
```
[the remark this answers is not in the document]
Comment (DirectorSloan) [reply easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb:13]: Hi @Micket, thx for looking over. Nethertheless I've started with setting `cstd` to `c++17` via toolchainopts first but it don't work for the build mech for this software. Did it work on your platform?

File: easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb (hunk 1 above)
Comment (DirectorSloan) [reply easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb:27]: You're absolutely right. Okay, thanks, I'll try

Comment (DirectorSloan) [reply easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb:27]: yikes, the new download path of URANOS doesn't contain a `src` directory just binary releases.

**Comment by DirectorSloan** [comment]
Sorry to say that I don't understand the error which the linter now gives :(

File: easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb (hunk 1 above)
Comment (DirectorSloan) [reply easybuild/easyconfigs/u/URANOS/URANOS-1.07-foss-2020b.eb]: Can be resolved. With new [commit](090c7920de204b7f0968d486634c50d1efb7bdf1) it works.

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**